validation/td-json-schema-validation.json

```diff
@@ -60,7 +60,8 @@
         "thing-context-w3c-uri": {
             "type": "string",
             "enum": [
-                "https://www.w3.org/2019/wot/td/v1"
+                "https://www.w3.org/2019/wot/td/v1",
```

Comment:
we should only have the "http://www.w3.org/ns/td" in the list since this schema should cover 1.1 version only